src/components/App/App.js

```diff
@@ -14,8 +15,8 @@ export default class App extends Component {
       (movie, index) => <p key={index}>{movie.title}</p>);
     return (
       <div>
-        <h1>Rotten Tomatillos</h1>
-        {this.state.movies.length ? titles : 
+        <Header />
+        {this.state.movies.length ? titles :
```

Comment:
I personally think the ternary is suitable here as it handles a solid `if...else` case.